tests/system/common.py

```diff
@@ -26,7 +27,6 @@
 from shakedown.dcos.package import install_package_and_wait, package_installed
 from shakedown.dcos.service import get_marathon_tasks, get_service_ips, get_service_task, service_available_predicate
 from shakedown.errors import DCOSException, DCOSHTTPException
```

Comment:
I wonder why we still need those two exception wrappers.

Reply:
We probably should remove them. Once [epic MARATHON-8101](https://jira.mesosphere.com/browse/MARATHON-8101) is done we can do a little cleanup and unify the clients. I think about the pattern that boto3 is using for AWS.